cypress_test/plugins/index.js

```diff
@@ -97,7 +96,7 @@ function pickTests(filename, foundTests) {
 	}
 
 	if (process.env.USER_INTERFACE === 'notebookbar') {
-		if (!isNotebookbarTest(filename,whitelists.notebookbarOnlyList)) {
+		if (!isNotebookbarTest(filename,blacklists.notebookbarOnlyList)) {
```

Comment:
it is preferred to not use words like blacklist/whitelist: https://tools.ietf.org/id/draft-knodel-terminology-00.html